packages/calcite-components/src/components/shell-panel/shell-panel.tsx

```diff
@@ -216,6 +227,19 @@ export class ShellPanel implements ConditionalSlotComponent, LocalizedComponent,
 
   @State() hasHeader = false;
 
+  // --------------------------------------------------------------------------
+  //
+  //  Events
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * @internal
+   */
+  @Event({ cancelable: false }) calciteInternalShellPanelResize: EventEmitter<{
+    resizing: boolean;
```

Comment:
Nitpick: having a `resizing` boolean on a resize event sticks out to me. WDYT of `state: "start" | "end"`, `status: "start" | "end"` or something similar? I'll leave this up to you.

Reply:
Yeah that's fine. Its internal so it doesn't really matter too much.

Reply:
Ill just use two events so its clearer.